### DISCUSSION TITLE
Print Input-File Variable in Console

### DISCUSSION THREAD

**Discussion by jmeier** [idaholab/moose#26667] [Q&A General] [answered]
Dear Moose-Community,

in a Moose-Input file one can define variables (e.g. by just writing  `bla = "blubb"`). Is there a way to to print the name of the variable and its value to the console? 

For now I'm using a function as shown below and `--show-input`. But is there a sane way to do this?
```
[Functions]
	[FakeUser_VariableBla]
		type = ParsedFunction
		expression = 'a'
		symbol_names = 'a'
		symbol_values = '1'
		control_tags = ${bla}
	[]
[]
```

**Comment by GiudGiud** [comment, marked as answer]
Hello

So you want to make the parser more verbose? I dont think there is an option.
If there was or if we added one, you would add it to the `[Debug]` parameter show_parser (to set to true)

Guillaume

**Reply by jmeier** [reply]
Dear Guillaume

Yes, exactly. I was looking for this because of https://github.com/idaholab/moose/issues/26664.

Thanks & Kind regards, Jörg

**Reply by GiudGiud** [reply]
Is https://github.com/idaholab/moose/issues/26664 fixed?

My guess on this one is that there is a missing bracket / quote. Please ping me there if you dont find the problem

**Reply by jmeier** [reply]
Dear Guillaume,

No, https://github.com/idaholab/moose/issues/26664 is not fixed up until now. And till now, I did not find a mistake (like a missing bracket).

Kind regards, Jörg

**Reply by GiudGiud** [reply]
ok let s discuss there
